case_studies/new_detection_encoder/config/config.yaml

```diff
@@ -0,0 +1,163 @@
+        n_tiles_h: 8
+        n_tiles_w: 8
+        n_batches: 128
+        batch_size: 64
+        valid_n_batches: 50
+        generate_device: "cpu"
+        num_workers: 5
+        fix_validation_set: true
+
+models:
+    decoder:
+        _target_: bliss.models.decoder.ImageDecoder
+        n_bands: 1
+        tile_slen: 4
+        ptile_slen: 52
+        border_padding: 24
+        psf_slen: 25
+        sdss_bands:
+            - 2
+        psf_params_file: ${paths.data}/sdss/94/1/12/psField-000094-1-0012.fits
+
+    detection_encoder:
+        _target_: case_studies.new_detection_encoder.main.NewDetectionEncoder
+        #  _target_: bliss.models.detection_encoder.DetectionEncoder
+        input_transform:
+            _target_: bliss.models.detection_encoder.ConcatBackgroundTransform
+        n_bands: ${models.decoder.n_bands}
+        tile_slen: ${models.decoder.tile_slen}
+        ptile_slen: 52
+        max_detections: ${models.prior.max_sources}
+        channel: 8
+        spatial_dropout: 0.0
+        dropout: 0.0
+        hidden: 128
+        annotate_probs: true
+        slack: 1.
+
+        optimizer_params:
+            lr: 1e-3
+        scheduler_params:
+            milestones: [8, 24]
+            gamma: 0.1
+
+        # this architecture is based on yolov5l.yaml, see
+        # https://github.com/ultralytics/yolov5/blob/master/models/yolov5l.yaml
+        architecture:
+            nc : 3
+            depth_multiple: 1.0  # model depth multiple
+            width_multiple: 1.0  # layer channel multiple
+            anchors:
+                - [4, 4]  # P3/8
+
+            backbone: [
+                # [from, number, module, args]
+                [-1, 1, Conv, [64, 5, 1]],  # 0-P1/2
+                [-1, 3, Conv, [64, 1, 1]],  # 1-P2/4
+                [-1, 1, Conv, [128, 3, 2]],  # 1-P2/4
+                [-1, 1, Conv, [128, 3, 1]],  # 1-P2/4
+                [-1, 1, Conv, [256, 3, 2]],  # 3-P3/8
+                [-1, 6, C3, [256]],
+                [-1, 1, Conv, [512, 3, 2]],  # 5-P4/16
+                [-1, 9, C3, [512]],
+                [-1, 1, Conv, [1024, 3, 2]],  # 7-P5/32
+                [-1, 3, C3, [1024]],
+                [-1, 1, SPPF, [1024, 5]],  # 9
+            ]
+
+            head: [
+                [-1, 1, Conv, [512, 1, 1]],
+                [-1, 1, nn.Upsample, [None, 2, 'nearest']],
+                [[-1, 6], 1, Concat, [1]],  # cat backbone P4
+                [-1, 3, C3, [512, false]],  # 13
+
+                [-1, 1, Conv, [256, 1, 1]],
+                [-1, 1, nn.Upsample, [None, 2, 'nearest']],
+                [[-1, 4, 5], 1, Concat, [1]],  # cat backbone P3
+                [-1, 3, C3, [256, false]],  # 17 (P3/8-small)
+
+                [[17], 1, Detect, [nc, anchors]],  # Detect(P3, P4, P5)
+            ]
+    prior:
+        _target_: bliss.models.prior.ImagePrior
+        n_bands: 1
+        max_sources: 1
+        mean_sources: 0.6931471805599453  # 50% no source, 50% one source
```

Comment:
It is more like m2. I just picked it because I thought it would train faster than if most tiles were empty. But indeed, we should test the new encoder in more realistic settings too.